_data/events.yml

```diff
@@ -435,10 +435,46 @@
   githubUrl: "https://github.com/itenium-be/React-ComponentProfiling"
   blogPost: ""
 
+- name: Story Telling and Feedback
+  subTitle:
+  date: 2023-02-02
+  summary: >
+    Finding the right words and finding the perfect way to say it really makes a big difference in every collaboration.
+  track: "FrontendTrack"
```

Comment:
It's not FrontendTrack for sure
Probably need a new track for these events
Maybe keep it empty for now...